### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by djdanielsson**
ran ansible-lint --fix to see what it would suggest changing

### CODE REVIEW

**Review by Tompage1994 — APPROVED** [review]
Sure...

Not been through it all but it seems sensible enough

**Comment by djdanielsson** [comment]
main ones I want to fix are the line length ones cuz I hate those

**Review by sean-m-sullivan — APPROVED** [review]

**Comment by djdanielsson** [comment]
ok I updated this let me know what you think and if we should merge it or not

**Review by sean-m-sullivan — APPROVED** [review]

**Comment by sean-m-sullivan** [comment]
I'm for the merge, needs to be rebased and I expect that to be hell ><

File: playbooks/set_organization.yaml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
       register: __list_files
 
-    - name: "Update the organization into the found files"
+    - name: Update the organization into the found files
       ansible.builtin.replace:
         path: "{{ __file.path }}"
-        regexp: '^(.*)organization: (.*)$'
-        replace: '\1organization: {{ new_organization }}'
+        regexp: "^(.*)organization: (.*)$"
+        replace: "\\1organization: {{ new_organization }}"
```
Comment (djdanielsson) [inline playbooks/set_organization.yaml:18]: I wonder if this will break something

**Review by sean-m-sullivan — APPROVED** [review]

**Pull request merged**